src/scss/_media.scss

```diff
@@ -0,0 +1,46 @@
+$phone-large-width: 25em; //400px
+$tablet-small-width: 40em; //640px
+$tablet-large-width: 47em; //752px
+$desktop-small-width: 60em; //960px;
+$desktop-width: 77em; //1200px
+$desktop-large-width: 95em; //1520px
+
+//Size that grid elements are contained at
+$contain: 75em; //1200px
+$contain-nav: 87.5em; //1400px
+
+@mixin phone-large {
```

Comment:
I highly recommend not setting up mixins to do this work. It is perfectly alright to use normal media queries in your SCSS files. This sets a lot of rigidness in your code that limits it.

See Ethan’s article on the issues with naming conventions like this: https://seesparkbox.com/foundry/using_names_to_manage_dozens_of_breakpoints